lib/tasks/cognigy/index.js

```diff
@@ -184,19 +252,57 @@ class Cognigy extends Task {
       this.performHook(cs, this.eventHook, {event: 'botMessage', message: evt})
         .then((redirected) => {
           if (redirected) {
-            this.logger.info('Cognigy_onTranscription: event handler for bot message redirected us to new webhook');
+            this.logger.info('Cognigy_onBotUtterance: event handler for bot message redirected us to new webhook');
             this.reportedFinalAction = true;
             this.performAction({cognigyResult: 'redirect'}, false);
           }
           return;
         })
         .catch(({err}) => {
-          this.logger.info({err}, 'Cognigy_onTranscription: error sending event hook');
+          this.logger.info({err}, 'Cognigy_onBotUtterance: error sending event hook');
         });
     }
+
     const text = parseBotText(evt);
     if (evt.data) this.config.update(evt.data);
-    if (text) this.prompts.push(text);
+    if (text) {
+      this._enqueueTask(async() => {
+        this.logger.info({text}, 'received text');
+        const sayTask = this._makeSayTask(text);
+        await sayTask.exec(cs, ep, this);
+      });
+    }
+
+
+    if (evt && evt.data && evt.data.type) {
```

Comment:
possible to remove this and just reference evt?.data?.type in the switch below?

Reply:
Sure